src/js/datadog-docs.js

```diff
@@ -171,57 +171,60 @@ $(document).ready(function () {
     headings.init();
   }
 
-  var client = algoliasearch("EOIG7V0A2O", 'bf60de88836cb62a73509ef075542065');
-  var results = new RegExp('[\?&]' + "s" + '=([^&#]*)').exec(window.location.href);
-  var $pagination = $('#tipue_search_content');
-  var query = results[1] || "*";
-
-  // get results from algolia
-  client.search([{
-    indexName: 'docs_english',
-    query: decodeURIComponent(query),
-    params: {
-      hitsPerPage: 200,
-      attributesToRetrieve: "*"
-    }
-  }], function (err, results) {
-    if (!err) {
-      // format and populate results
-      $('#tipue_search_input').val(decodeURIComponent(query));
-      var hits = results['results'][0]['hits'];
-      var formatted_results = "";
-      if (hits.length) {
-        $('#tipue_search_content').prepend('<div id="tipue_search_results_count">' + hits.length + ' results</div>');
-        for (var i in hits) {
-
-          var hit = hits[i];
-                console.log(hit);
-          formatted_results += '<div class="hit">';
-          formatted_results += '<div class="tipue_search_content_title">' +
-            '<a href="' + hit["URL"] + '">' + hit["title"] + '</a></div>';
-          formatted_results += '<div class="tipue_search_content_url">' +
-            '<a href="' + hit["URL"] + '">' + hit["URL"].replace('https://docs.datadoghq.com', '') + '</a></div>';
-          var text = hit.page_description;
-          formatted_results += '<div class="tipue_search_content_text">' +
-            text + '</div>';
-          formatted_results += '</div>';
-        }
-      } else {
-        $('#tipue_search_content').prepend('<div id="tipue_search_results_count">' + hits.length + ' results</div>');
-      }
-      $('#tipue_search_content .content').html(formatted_results);
-
-      // load pagination
-      $('#tipue_search_content').pajinate({
-        num_page_links_to_display : 9,
-        items_per_page : 7,
-        wrap_around: false,
-        show_first_last: false
-      });
-
-    }
-  });
+  // algolia
+  if (window.location.href.indexOf('/search/') > -1) {
```

Comment:
only execute this code on the search page.

Reply:
👍